build.sbt

```diff
@@ -10,7 +10,7 @@ scalaVersion := "2.11.6"
 
 val sprayV = "1.3.2"
 val DowngradedSprayV = "1.3.1"
-val akkaV = "2.3.6"
+val akkaV = "2.3.9"
```

Comment:
misread the download page. I knew there was a new release and just looked to see what it was, will update